### PR TITLE
Tech note: Create credentials for the Command Generator

### PR DESCRIPTION
**Pull request by thenets**

### CODE REVIEW

File: stories/topics/proc-create-credentials-cmd-generator.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
+[id="prod-create-credentials-cmd-generator"]
+
+= Create credentials for the Command Generator
+
+It is required to have an AWS credentials file for all day-2 operations, like adding and removing Extension Nodes.
```
Comment (ianf77) [inline stories/topics/proc-create-credentials-cmd-generator.adoc:5]: You must have an AWS ... , such as adding and removing ...

File: stories/topics/proc-create-credentials-cmd-generator.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+The file must follow the format of the AWS credentials file for "Long-term credentials", which includes the `aws_access_key_id` and `aws_secret_access_key` variables. For the official AWS documentation, read link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[AWS Command Line Interface User Guide].
```
Comment (ianf77) [inline stories/topics/proc-create-credentials-cmd-generator.adoc:7]: For further information, read link...... in the AWS documentation.

File: stories/topics/proc-create-credentials-cmd-generator.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+An example of the credentials file format is:
+
+[NOTE]
+====
+It is required to use the `default` profile name in the credentials file. Any other profile name will not be recognized by the Command Generator. For more information, read xref:con-tech-note-cmd-generator-aws-credentials[Command generator - AWS Credentials File] technical note.
```
Comment (ianf77) [inline stories/topics/proc-create-credentials-cmd-generator.adoc:13]: You must use the ` default `...

The Command generator does not recognize any other name. 

read the ..... technical note.

File: stories/topics/proc-create-credentials-cmd-generator.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+====
+
+[literal, options="nowrap" subs="+attributes"]
+----
+[default]
+aws_access_key_id=AKIAIOSFODNN7EXAMPLE
+aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+----
+
+After the credentials file is created, you can use it in the command generator by passing the file path to the `-c` option. Example:
```
Comment (ianf77) [inline stories/topics/proc-create-credentials-cmd-generator.adoc:23]: option, for example:

File: stories/topics/proc-create-credentials-cmd-generator.adoc [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+/home/user/project/extra_vars.yaml:
```
Comment (ianf77) [inline stories/topics/proc-create-credentials-cmd-generator.adoc:25]: Do you simply enter the path? Or is there an associated command?

**Review by ianf77 — COMMENTED** [review]
A few comments

File: stories/topics/proc-create-credentials-cmd-generator.adoc (hunk 5 above)
Comment (thenets) [reply stories/topics/proc-create-credentials-cmd-generator.adoc]: I did some changes in the format.

File: stories/topics/proc-create-credentials-cmd-generator.adoc [hunk 6]
```diff
@@ -0,0 +1,59 @@
+[id="prod-create-credentials-cmd-generator"]
+
+= Create credentials for the Command Generator
+
+You must have an AWS credentials file for all day-2 operations, such as adding and removing Extension Nodes.
+
+The file must follow the format of the AWS credentials file for "Long-term credentials", which includes the `aws_access_key_id` and `aws_secret_access_key` variables. For further information, read link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[AWS Command Line Interface User Guide] in the AWS documentation.
+
+An example of the credentials file format is:
```
Comment (ianf77) [inline stories/topics/proc-create-credentials-cmd-generator.adoc:9]: I would swap the line 'An example of  and the following NOTE, it looks odd saying the format is: and then having a note.

**Review by ianf77 — COMMENTED** [review]
One comment

File: stories/topics/proc-create-credentials-cmd-generator.adoc (hunk 6 above)
Comment (thenets) [reply stories/topics/proc-create-credentials-cmd-generator.adoc:9]: did it

**Comment by thenets** [comment]
rebased

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**